<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wyatt's Simple Flashcard App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="deckListWidget">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>40</y>
      <width>391</width>
      <height>281</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addDeckButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyDeckButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runDeckButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Opens a flashcard slideshow that allows you to test yourself.</string>
    </property>
    <property name="text">
     <string>Run Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCardDeckButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renameDeckButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>230</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Rename Deck</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runFlippedDeckButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>271</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>All flashcards will be flipped</string>
    </property>
    <property name="text">
     <string>Run Deck (flipped)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to_Use"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHow_to_Use">
   <property name="text">
    <string>How to Use</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
